.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
please remove this and all the yarn.lock changes, you don't need to change those files. We use yarn v1